### PR TITLE
format route folder

### PR DESCRIPTION
**Pull request by saurabhketech**

### CODE REVIEW

File: src/routes/fetch_mail.js [hunk 1]
```diff
@@ -3,25 +3,25 @@ import auth from "../middleware/auth";
...
-	app.route("/email/countEmail").get(auth.requiresLogin, fetch_email.countEmail);
+    app.route("/email/countEmail").get(auth.requiresLogin, fetch_email.countEmail);
 
     /* Route for assign Multiple Tag  */
-	app.route("/email/assignMultiple").post(auth.requiresLogin, fetch_email.assignMultiple);
+    app.route("/email/assignMultiple").post(auth.requiresLogin, fetch_email.assignMultiple);
 
     /* Route for delete Tag  */
-	app.route("/email/deleteTag").post(auth.requiresLogin, fetch_email.deleteTag);
+    app.route("/email/deleteTag").post(auth.requiresLogin, fetch_email.deleteTag);
```
Comment (arun-kumar-codes) [inline src/routes/fetch_mail.js:18]: this should be of type delete()

File: src/routes/fetch_mail.js [hunk 2]
```diff
@@ -3,25 +3,25 @@ import auth from "../middleware/auth";
...
 
     /* Route for change unread status  */
-	app.route("/email/changeUnreadStatus").post(auth.requiresLogin, fetch_email.changeUnreadStatus);
+    app.route("/email/changeUnreadStatus").post(auth.requiresLogin, fetch_email.changeUnreadStatus);
 
-	/* Route for delete email  */
-	app.route("/email/deleteEmail").post(auth.requiresLogin, fetch_email.deleteEmail);
+    /* Route for delete email  */
+    app.route("/email/deleteEmail").post(auth.requiresLogin, fetch_email.deleteEmail);
```
Comment (arun-kumar-codes) [inline src/routes/fetch_mail.js:24]: this should be of type delete()

File: src/routes/inbox.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
 import inbox from "../controllers/inbox";
-// import auth from "../middleware/auth";
 
 export default (app) => {
     /* Route for get INBOX */
-	app.route("/email/inbox/:page").get( inbox.getInbox);
+    app.route("/email/inbox/:page").get(inbox.getInbox);
```
Comment (arun-kumar-codes) [inline src/routes/inbox.js:5]: this should requireLogin

File: src/routes/inbox.js [hunk 4]
```diff
@@ -1,12 +1,11 @@
...
 
     /* Route for fetch by UID*/
-	app.route("/email/:uid").get( inbox.getUid);
+    app.route("/email/:uid").get(inbox.getUid);
```
Comment (arun-kumar-codes) [inline src/routes/inbox.js:8]: this should require login

File: src/routes/imap.js [hunk 5]
```diff
@@ -1,24 +1,23 @@
...
-// import baseController from '../controllers/BaseAPIController'
 
 export default (app) => {
     /* Route for imap save  */
-	app.route("/imap/save").post(auth.requiresAdmin, imap.save);
+    app.route("/imap/save").post(auth.requiresAdmin, imap.save);
 
     /* Route for imap update  */
-	app.route("/imap/update/:id").put(auth.requiresAdmin, imap.update);
+    app.route("/imap/update/:id").put(auth.requiresAdmin, imap.update);
```
Comment (arun-kumar-codes) [inline src/routes/imap.js:9]: change :id to :imap_id everywhere

File: src/routes/fetch_mail.js [hunk 6]
```diff
@@ -3,25 +3,25 @@ import auth from "../middleware/auth";
...
-	app.route("/email/assignTag").post(auth.requiresLogin, fetch_email.assignTag);
+    app.route("/email/assignTag").post(auth.requiresLogin, fetch_email.assignTag);
 
     /* Route for count emails on the basis of tag */
-	app.route("/email/countEmail").get(auth.requiresLogin, fetch_email.countEmail);
+    app.route("/email/countEmail").get(auth.requiresLogin, fetch_email.countEmail);
 
     /* Route for assign Multiple Tag  */
-	app.route("/email/assignMultiple").post(auth.requiresLogin, fetch_email.assignMultiple);
+    app.route("/email/assignMultiple").post(auth.requiresLogin, fetch_email.assignMultiple);
```
Comment (arun-kumar-codes) [inline src/routes/fetch_mail.js:15]: all these routes should have :email_id

File: src/routes/fetch_mail.js [hunk 7]
```diff
@@ -3,25 +3,25 @@ import auth from "../middleware/auth";
 
 export default (app) => {
     /* Route for fetch email from mongoDb  */
-	app.route("/email/fetch").post(auth.requiresLogin, fetch_email.fetch);
+    app.route("/email/fetch").post(auth.requiresLogin, fetch_email.fetch);
 
     /* Route for add tag  */
-	app.route("/email/assignTag").post(auth.requiresLogin, fetch_email.assignTag);
+    app.route("/email/assignTag").post(auth.requiresLogin, fetch_email.assignTag);
```
Comment (arun-kumar-codes) [inline src/routes/fetch_mail.js:9]: all these routes should have :email_id

File: src/routes/fetch_mail.js (hunk 2 above)
Comment (arun-kumar-codes) [inline src/routes/fetch_mail.js:24]: all these routes should have :email_id

File: src/routes/fetch_mail.js [hunk 8]
```diff
@@ -3,25 +3,25 @@ import auth from "../middleware/auth";
...
-	app.route("/email/assignMultiple").post(auth.requiresLogin, fetch_email.assignMultiple);
+    app.route("/email/assignMultiple").post(auth.requiresLogin, fetch_email.assignMultiple);
 
     /* Route for delete Tag  */
-	app.route("/email/deleteTag").post(auth.requiresLogin, fetch_email.deleteTag);
+    app.route("/email/deleteTag").post(auth.requiresLogin, fetch_email.deleteTag);
 
     /* Route for change unread status  */
-	app.route("/email/changeUnreadStatus").post(auth.requiresLogin, fetch_email.changeUnreadStatus);
+    app.route("/email/changeUnreadStatus").post(auth.requiresLogin, fetch_email.changeUnreadStatus);
```
Comment (arun-kumar-codes) [inline src/routes/fetch_mail.js:21]: all these routes should have :email_id

File: src/routes/imap.js [hunk 9]
```diff
@@ -1,24 +1,23 @@
...
+    app.route("/imap/delete/:id").delete(auth.requiresAdmin, imap.deleteImap);
 
     /* Route for fetch Imap Data */
-	app.route("/imap/get/:page").get(auth.requiresAdmin, imap.getImap);
+    app.route("/imap/get/:page").get(auth.requiresAdmin, imap.getImap);
 
-	/* Route for fetch Imap Data */
-	app.route("/imap/statusActive").post(auth.requiresAdmin, imap.statusActive);
+    /* Route for fetch Imap Data */
+    app.route("/imap/statusActive").post(auth.requiresAdmin, imap.statusActive);
```
Comment (arun-kumar-codes) [inline src/routes/imap.js:18]: should have :imap_id

File: src/routes/smtp.js [hunk 10]
```diff
@@ -3,27 +3,27 @@ import auth from "../middleware/auth";
...
 
-	/* Route for  Smtp test */
-	app.route("/smtp/testSmtp").post(auth.requiresAdmin, smtp.testSmtp);
+    /* Route for  Smtp test */
+    app.route("/smtp/testSmtp").post(auth.requiresAdmin, smtp.testSmtp);
 
-	/* Route for  change status */
-	app.route("/smtp/changeStatus").post(auth.requiresAdmin, smtp.changeStatus);
+    /* Route for  change status */
+    app.route("/smtp/changeStatus").post(auth.requiresAdmin, smtp.changeStatus);
```
Comment (arun-kumar-codes) [inline src/routes/smtp.js:24]: this should have smtpId

File: src/routes/tempVariable.js [hunk 11]
```diff
@@ -3,16 +3,16 @@ import auth from "../middleware/auth";
```
Comment (arun-kumar-codes) [inline src/routes/tempVariable.js:3]: why is filename tempVariable.js ?

File: src/routes/template.js [hunk 12]
```diff
@@ -3,16 +3,16 @@ import auth from "../middleware/auth";
 
 export default (app) => {
     /* Route for Template Create  */
-	app.route("/template/add").post(auth.requiresAdminOrHr, template.create);
+    app.route("/template/add").post(auth.requiresAdminOrHr, template.create);
 
     /* Route for Template update  */
-	app.route("/template/update/:templateId").put(auth.requiresAdminOrHr, template.update);
+    app.route("/template/update/:templateId").put(auth.requiresAdminOrHr, template.update);
```
Comment (arun-kumar-codes) [inline src/routes/template.js:9]: need to change in all routes, discuss with me. the routes are not done properly.

File: src/routes/token.js [hunk 13]
```diff
@@ -2,7 +2,7 @@ import auth from "../middleware/auth";
 
 export default (app) => {
     /* Route for Token Verification  */
-	app.route("/verify").get(auth.verifyToken);
+    app.route("/verify").get(auth.verifyToken);
```
Comment (arun-kumar-codes) [inline src/routes/token.js:5]: make this file auth.js

File: src/routes/user.js [hunk 14]
```diff
@@ -2,10 +2,10 @@ import user from "../controllers/user";
 
 export default (app) => {
     /* Route for User Registration  */
-	app.route("/user/register").post(user.create);
+    app.route("/user/register").post(user.create);
```
Comment (arun-kumar-codes) [inline src/routes/user.js:5]: move these routes to auth.js

File: src/routes/tempVariable.js (hunk 11 above)
Comment (saurabhketech) [reply src/routes/tempVariable.js:3]: name changed to variable.js

File: src/routes/token.js (hunk 13 above)
Comment (saurabhketech) [reply src/routes/token.js:5]: done

File: src/routes/user.js (hunk 14 above)
Comment (saurabhketech) [reply src/routes/user.js:5]: move routes to auth.js

File: src/routes/fetch_mail.js (hunk 7 above)
Comment (saurabhketech) [reply src/routes/fetch_mail.js:9]: done

File: src/routes/fetch_mail.js (hunk 6 above)
Comment (saurabhketech) [reply src/routes/fetch_mail.js:15]: done

File: src/routes/fetch_mail.js (hunk 1 above)
Comment (saurabhketech) [reply src/routes/fetch_mail.js:18]: done

File: src/routes/fetch_mail.js (hunk 8 above)
Comment (saurabhketech) [reply src/routes/fetch_mail.js:21]: done

File: src/routes/fetch_mail.js (hunk 2 above)
Comment (saurabhketech) [reply src/routes/fetch_mail.js:24]: done

Comment (saurabhketech) [reply src/routes/fetch_mail.js:24]: done

File: src/routes/inbox.js (hunk 3 above)
Comment (saurabhketech) [reply src/routes/inbox.js:5]: done

File: src/routes/inbox.js (hunk 4 above)
Comment (saurabhketech) [reply src/routes/inbox.js:8]: done

File: src/routes/imap.js (hunk 5 above)
Comment (saurabhketech) [reply src/routes/imap.js:9]: done

File: src/routes/imap.js (hunk 9 above)
Comment (saurabhketech) [reply src/routes/imap.js:18]: done

File: src/routes/smtp.js (hunk 10 above)
Comment (saurabhketech) [reply src/routes/smtp.js:24]: done

File: src/routes/template.js (hunk 12 above)
Comment (saurabhketech) [reply src/routes/template.js:9]: ok sir

**Pull request merged**